Observer/ConfigCredentialsChangeObserver.php

```
@@ -61,9 +61,9 @@ public function execute(Observer $observer)
             try {
                 $this->credentials->getToken();
                 $this->credentials->deleteToken();
-                $this->messageManager->addNoticeMessage(__('You have changed credentials to Allegro account. Your current connection has been lost and you have to connect with Allegro account again'));
+                $this->messageManager->addNoticeMessage(__('You have changed credentials to Allegro account. Your current connection has been lost and you have to connect with Allegro account again'));//phpcs:ignore
```

why suppressing the error?